src/main/java/org/mockbukkit/mockbukkit/block/BlockMock.java

```diff
@@ -695,4 +693,9 @@ public boolean canPlace(@NotNull BlockData data)
 		throw new UnimplementedOperationException();
 	}
 
+	public void setDrops(Collection<ItemStack> items)
+	{
+		drops = items;
```

Comment:
Probably a good idea to copy this to a list

Reply:
Copied.